build.gradle

```diff
@@ -60,8 +61,8 @@ repositories {
     }
```

Comment:
Don't forget to update the forceLinkClasses in the ios section at the bottom to include the changed package name there as well.